docs/getting-started-guides/coreos/bare_metal_offline.md

```diff
@@ -42,154 +42,134 @@ To setup CentOS PXELINUX environment there is a complete [guide here](http://doc
 
 1. Install packages needed on CentOS
 
-```shell
-sudo yum install tftp-server dhcp syslinux
-```
+        sudo yum install tftp-server dhcp syslinux
 
 2. `vi /etc/xinetd.d/tftp` to enable tftp service and change disable to 'no'
 
-```conf
-disable = no
-```
+        go get github.com/vmware/photon-controller-cli/photon
```

Comment:
I suspect this is a copy paste error - photon doesnt appear to be relevant to the CoreOS installation...

Reply:
Oh, yes. Thanks.